csharp/test/Microsoft.ML.OnnxRuntime.Tests.Common/OrtEnvTests.cs

```diff
@@ -172,13 +170,15 @@ public void TesEnvWithCustomLogger()
 
             var model = TestDataLoader.LoadModelFromEmbeddedResource("squeezenet.onnx");
             // Trigger some logging
-            using (var session = new InferenceSession(model)) ;
+            // Empty stmt intentional
```

Comment:
nit: better if comments explain 'why' rather than just 'what'. I assume it's because what you're testing just requires session initialization, so a comment to that effect would help the next developer more.